docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  cinema:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 127.0.0.1:8000"

```

Comment:
The `runserver` command is binding to `127.0.0.1:8000`, which means it will only be accessible from within the container. Consider binding to `0.0.0.0:8000` to make the server accessible from outside the container.